src/Symfony/Bundle/TwigBundle/TwigEngine.php

```diff
@@ -51,7 +51,8 @@ public function setDefaultEscapingStrategy($strategy)
     }
 
     /**
-     * @deprecated Deprecated since version 2.3, to be removed in 3.0. Use TwigDefaultEscapingStrategy instead.
+     * @deprecated since version 2.3, to be removed in 3.0.
```

Comment:
Maybe we should mark it as deprecated from 2.7 only (given it is the first minor version which will have the new method from the start, even though the method is not used by Symfony even in older versions.